sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -340,6 +340,11 @@ static void _send_to_iface(kernel_pid_t iface, gnrc_pktsnip_t *pkt)
         gnrc_pktbuf_release(pkt);
         return;
     }
+#ifdef MODULE_NETSTATS_IPV6
+    if_entry->stats.tx_success++;
+    if_entry->stats.tx_bytes += gnrc_pkt_len(pkt->next);
```

Comment:
Hm, no idea. In [gnrc_pktbuf_static.c:423](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/pktbuf_static/gnrc_pktbuf_static.c#L423) the size for a `pktnsip` gets aligned, but I have no idea where the magic happens that `gnrc_pkt_len` reports the correct size. Maybe in [gnrc_pktbuf_static.c:449](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/pktbuf_static/gnrc_pktbuf_static.c#L449)?